src/fabric-1.19.2/templates/modbase/mixins.json.ftl

```diff
@@ -4,9 +4,11 @@
   "package": "${package}.mixins",
   "compatibilityLevel": "JAVA_17",
   "mixins": [
-	"EntityMixin",
+	"${settings.getJavaModName()}RepairItemRecipeMixin"<#if w.hasToolsOfType("Fishing rod")>,
```

Comment:
It should be the item base conditio , no?

Reply:
Mixins.json already has the item base condition to be created, which is why it isn't present here